Source: Elias Pipping <pipping@exherbo.org>
Upstream: Submitted
Reason: otfview should not be built when --without-x is specified (fails to link)
--- libotf-0.9.12/configure.ac
+++ libotf-0.9.12/configure.ac
@@ -40,6 +40,8 @@
 
 AC_CHECK_HEADERS([X11/Xaw/Command.h], , no_x=yes)
 
+AM_CONDITIONAL([X11_UNAVAILABLE], [test x$no_x = xyes])
+
 # Checks for typedefs, structures, and compiler characteristics.
 
 # Checks for library functions.
diff -ur libotf-0.9.12//example/Makefile.am /home/pipping/libotf-0.9.12//example/Makefile.am
--- libotf-0.9.12/example/Makefile.am
+++ libotf-0.9.12/example/Makefile.am
@@ -22,7 +22,11 @@
 
 ## Process this file with Automake to create Makefile.in
 
-bin_PROGRAMS = otfdump otfview otflist otftobdf
+bin_PROGRAMS = otfdump otflist otftobdf
+
+if !X11_UNAVAILABLE
+bin_PROGRAMS += otfview
+endif
 
 CommonLDADD = ${top_builddir}/src/libotf.la
 
